### ISSUE TITLE
update sdk 50

### ISSUE THREAD

**Issue by ghost** [EvanBacon/expo-router-layouts-example#9]
please update expo router to v3

**Comment by josiext** [comment]
Yep, copying the same exact code to an app with expo router v3 make Bottom Sheet code stop working.. 

![image](https://github.com/EvanBacon/expo-router-layouts-example/assets/79015124/8c2f8800-2653-4bc7-929d-632130333189)
